TestUrdfImporter/Assets/Readme.asset

```diff
@@ -0,0 +1,50 @@
+%YAML 1.1
```

Comment:
What is this file for? It seems that Github displays this file as normal README.md and it is not in a friendly format to read.

Reply:
This is the tutorial Inspector window for a new URP project that comes included by default I believe!

Reply:
Ah, I see! thank you.

But this page looks weird on Github https://github.com/Unity-Technologies/URDF-Importer/tree/devin/to-yamato/TestUrdfImporter/Assets. Is it possible to modify the name of the file so that we don't have so much raw code displaying on the Github page? Just for aesthetics.

Reply:
I don't think we can modify the file itself much, but we could modify our .gitattributes to treat .asset files as binary and push them into LFS instead - I think that would result in any .asset files simply not displaying anything in the browser view.